omod/src/main/webapp/resources/jsonforms/HMIS-HIV-003-HivCareArtCard-ClinicalAssessmentPage.json

```diff
@@ -1113,6 +1113,9 @@
                 "rendering": "select",
                 "programUuid": "18c6d4aa-0a36-11e7-8dbb-507b9dc4c741",
                 "workflowUuid": "306c02b7-198e-4066-8b2c-39e40a1bcc53"
+              },
```

Comment:
@slubwama did checkout this too https://github.com/METS-Programme/openmrs-module-ugandaemr/pull/562

Reply:
@Mwanje   not I didn't But looks like you are adding skip logic to the DSDM field. Let me see if your approach works and integrate it here. Then close your PR since This covers it and more

Reply:
@Mwanje  I have tried using your logic to hiding and showing the same field but doesn't seem to work as expected.